core/assets/bundles/bundle_fr.properties

```diff
@@ -1,23 +1,23 @@
 credits.text = Créé par [ROYAL]Anuken[] - [SKY]anukendev@gmail.com[]\n\n[GRAY]
 credits = Crédits
-contributors = Traducteurs et contributeurs
-discord = Rejoignez le Discord de Mindustry
-link.discord.description = Le discord officiel de Mindustry!
+contributors = Traducteurs et Contributeurs
+discord = Rejoignez le Discord de Mindustry !
+link.discord.description = Le Discord officiel de Mindustry!
 link.reddit.description = Le subreddit de Mindustry
 link.github.description = Code source du jeu
 link.changelog.description = Liste des mises a jour
 link.dev-builds.description = Versions instables du jeu
 link.trello.description = Trello officiel pour les ajouts futurs
-link.itch.io.description = Page itch.io avec lien de téléchargement pour PC
+link.itch.io.description = Page itch.io avec les téléchargements pour PC
 link.google-play.description = Google Play Store
 link.f-droid.description = Catalogue F-Droid
 link.wiki.description = Le wiki officiel de Mindustry
-linkfail = Erreur lors de l'ouverture du lien !\nL'URL a été copiée dans votre presse papier.
+linkfail = Erreur lors de l'ouverture du lien !\nL'URL a été copiée dans votre presse-papiers.
```

Comment:
```suggestion
linkfail = Erreur lors de l'ouverture du lien !\nL'URL a été copiée dans votre presse-papier.
```

Reply:
According to [wikipedia](https://fr.wikipedia.org/wiki/Presse-papier_(informatique)), presse-papiers is the older writing.